### ISSUE TITLE
Proposal for rasdaman jupyter notebook template

### ISSUE THREAD

**Issue by ocampos16** [FAIRiCUBE/uc2-agriculture-biodiversity-nexus#5]
I have updated the repo with the newest version of the jupyter notebook for the [rasdaman ML UDF proof of concept](https://github.com/FAIRiCUBE/uc2-agriculture-biodiversity-nexus/blob/main/rasdaman-ml-udf/proof_of_concept/FAIRICUBE%20Machine%20Learning%20UDF%20Proof%20of%20Concept.ipynb). This will be the endpoint for the [Resource metadata](https://github.com/FAIRiCUBE/resource-metadata/issues/7).

It would be nice if we agree on the structure of this notebook so that it can later be used as a template for the other ML UDFs. Maybe even programmatically generating the preamble of the notebook with the metadata provided by [Resource metadata](https://github.com/FAIRiCUBE/resource-metadata/issues/7).

**Comment by robknapen** [comment]
Indeed, good to streamline it a bit and clarify what should go into the resource metadata, a reference notebook (if available) and an example (also a notebook)? And how one can automatically be created from another, otherwise it will be a lot of work to fill everything in and keep it in sync (as there might be multiple variants and versions of models).

**Comment by robknapen** [comment]
Maybe this should be moved to the central FAIRiCUBE Hub issue list?

**Comment by pebau** [comment]
my 2 cents:
- don't know what a reference notebook is. We planned to do just the example one.
- I do not believe we can automate this fully, data are just too heterogeneous and need different handling.
